woocommerce/woocommerce-ios
# Receipt print button unresponsive after first tap

Closes: #13575


## Description


In `ReceiptViewController` we already had a reference to `UIPrintInteractionController` which was configured once and on every next usage of it we had to configure it again.

## Testing information


- Go to the Orders tab.
- Select an order that has been paid for.
- Tap `See Receipt`.
- Tap the printer icon on the receipt.
- Tap `Cancel` or select the share icon and share the receipt.
- Tap the printer icon on the receipt **again**, and check that the printing UI shows again **properly**.

## Screenshots


| Before    | After |
| ----------- | ----------- |
| ![print before](https://github.com/user-attachments/assets/915e484a-5861-4dac-b1e9-c9863661978c)  | ![print after](https://github.com/user-attachments/assets/615eb0c3-0194-4060-9512-531fbb83f44e)    |


---
